### ISSUE TITLE
NaN as output if I use the + sign 

### ISSUE THREAD

**Issue by merlinuwe** [SilentVoid13/Templater#1015]
Hi,

when I use the plus sign

```
creation_date: <%+ tp.file.creation_date("dddd, DD. MMMM YYYY, HH:mm:ss") %> Uhr

modification_date: <%+ tp.file.last_modified_date("dddd, DD. MMMM YYYY, HH:mm:ss") %> Uhr

```

as mentioned here https://silentvoid13.github.io/Templater/commands/dynamic-command.html

I get NaN in the file made with this template.

Also:

# <%+ tp.file.title %>

does guve NaN.

Windows 10, latest plugins, latest obsidian.

**Comment by welpdx** [comment]
Couldn't replicate bug. Switching to Preview mode from Edit mode displays the correct result from the dynamic command.

Windows 11 Home 22H2
Obsidian v1.1.9 (No other plugin installed other than Templater)
Templater v1.16.0


![img](https://i.imgur.com/ygru7IJ.gif)

---

Ok I think I get your problem though. 
When ever you insert template or create a new note with template, the dynamic code is executed instead of simply copying the dynamic code over as text. 

Possibly the same issue as #910

**Comment by Zachatoo** [comment]
I highly recommend using dataview for this purpose instead, it works way more reliably for dynamic content, which is why this issue came up to deprecate dynamic commands at some point #913

**Comment by welpdx** [comment]
@Zachatoo Ah I see! That's why the function was nominated for deprecation!

**Comment by BarryBurton57** [comment]
> I highly recommend using dataview for this purpose instead, it works way more reliably for dynamic content, which is why this issue came up to deprecate dynamic commands at some point #913

How would you use dataview for this purpose though? Do you mean insert a dataview query in the template file?

**Comment by Zachatoo** [comment]
Yes, exactly. Other plugins tend to work great with Templater in this regard.

````js
<% tp.file.title %>
More of whatever...

```dataview
dataview stuff
```

Or inline dataview `= this.whatever`
````
